tutorials/text/text_intro.py

```diff
@@ -353,7 +353,6 @@
 #  Finally, we can specify functions for the formatter using
 # `matplotlib.ticker.FuncFormatter`.
 
-
```

Comment:
This line needs to stay to make PEP8 happy.

Reply:
I see. Thanks for reminding! And it seems that another blank line is also needed after this function definition.